Tests/RockLib.Messaging.Tests/ReloadingReceiverTests.cs

```diff
@@ -230,8 +226,9 @@ public void DisposeTest()
             mockChangeListener.Verify(m => m.Dispose());
         }
 
-        public class TestReceiverOptions
-        {
+#pragma warning disable CA1034 // Nested types should not be visible
+		public class TestReceiverOptions
+		{
```

Comment:
Looks like the indentation got messed up here.

Reply:
Fixed :)